README.md

```diff
@@ -1,45 +1,18 @@
 # VET CLINIC DATABASE
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you remove this title at the top of your readme file because you already have a title at the bottom of your Table of Contents 🤔 so there is no need to put this title at the top of your readme 😊 Kindly remove it to make your readme descriptive and professional 👍